test/NuGet.Core.FuncTests/NuGet.Packaging.FuncTest/SigningTests/AllowListVerificationProviderTests.cs

```diff
@@ -979,6 +982,98 @@ public AllowListVerificationProviderTests(SigningTestFixture fixture)
             }
         }
 
+        [CIOnlyFact]
+        public async Task GetTrustResultAsync_PackageSignedWithCertFromRepositoryCertificateList_SuccessAsync()
```

Comment:
Update the name to specify which type of signature it is the one that is in the package. Follow the convention on the other tests on the file:

`<method>_<type_of_signed_package>_<list_and_targets_of_entries>_<behavior>_<expected_result>`